src/shared/prompts.ts

```diff
@@ -1,8 +1,9 @@
 import * as prompts from "@clack/prompts";
 
-export function handleCancel(): never {
+export function handleCancel() {
 	prompts.cancel("Operation cancelled. Exiting - maybe another time? 👋");
-	process.exit(0);
+	// eslint-disable-next-line n/no-process-exit
```

Comment:
Seems `process.exit(1)` is important to terminate the CLI from this point onwards. So added eslint disable next line.